bindings/sundials4py/examples/arkode/ark_lotka_volterra_ASA.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+# -----------------------------------------------------------------
+# Programmer(s): Cody J. Balos
+# -----------------------------------------------------------------
+# Python port of the SUNDIALS ARKODE Lotka-Volterra adjoint
+# sensitivity example (ark_lotka_volterra_ASA.c)
+# -----------------------------------------------------------------
+import numpy as np
+import sundials4py.core as sun
+import sundials4py.arkode as ark
+
+
+class LotkaVolterraODE:
+    def __init__(self, p):
+        self.p = np.array(p, dtype=sun.sunrealtype)
+        self.NEQ = 2
+        self.NP = 4
+
+    def set_init_cond(self, yvec):
+        y = sun.N_VGetArrayPointer(yvec)
```

Comment:
Any particular reasons for using `sun.` rather than `import *` or just testing it out?

Reply:
I was testing it out. The python autocomplete in VS Code only works if you do this.

Reply:
I think its nice to have one example of this.